src/objects/Term_Grade__c.object

```diff
@@ -0,0 +1,233 @@
+    <enableActivities>true</enableActivities>
+    <enableBulkApi>true</enableBulkApi>
+    <enableFeeds>false</enableFeeds>
+    <enableHistory>false</enableHistory>
+    <enableLicensing>false</enableLicensing>
+    <enableReports>true</enableReports>
+    <enableSearch>true</enableSearch>
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Attempted_Credits__c</fullName>
+        <description>The number of credits attempted for this course and grading period.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The number of credits attempted for this course and grading period.</inlineHelpText>
+        <label>Attempted Credits</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>0</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Course_Connection__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The class in which the student is received this grade.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The class in which the student is received this grade.</inlineHelpText>
+        <label>Course Connection</label>
+        <referenceTo>Course_Enrollment__c</referenceTo>
+        <relationshipLabel>Term Grades</relationshipLabel>
+        <relationshipName>Term_Grades</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Earned_Credits__c</fullName>
+        <description>The number of credits earned for this course and grading period.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The number of credits earned for this course and grading period.</inlineHelpText>
+        <label>Earned Credits</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Letter_Grade__c</fullName>
+        <description>The letter grade the student earned for this course. For example, &quot;A+&quot; or any other text value.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The letter grade the student earned for this course. For example, &quot;A+&quot; or any other text value.</inlineHelpText>
+        <label>Letter Grade</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Numerical_Grade__c</fullName>
+        <description>The numerical grade the student earned for this course.</description>
```

Comment:
As a general note, it's not clear to me, even with the help text, what the difference between this field and Percent Grade is.

Reply:
I would defer to julie for help text, but in general, Numerical grade is usually out of 100 points. Very common for individual graded items, rubrics (K-12), etc.  e.g. 87.5/100 on Final Exam.  Sometimes it can even be out of 20, for example, Student Sophie got 18.5/20 on her final quiz. Percent grade is a % and can be calculated off Numerical sometimes.  We just have to cover both use-cases as different parts of the world, schools, verticals use diff. methods.

Reply:
@aligeez Sounds like you'd need a text field for your example of 18.5/20, or would they just be recording "18.5"?

Reply:
Looks like Julie updated this in the spec.